morse.rb

```diff
@@ -0,0 +1,30 @@
+def decode_char(char)
+  morse = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
+    '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J',
+    '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O',
+    '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T',
+    '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y',
+    '--..' => 'Z'
+  }
+  morse[char]
+end
+
+def decode_word(word)
+  new_str = ''
+  word.split.each do |char|
+    new_str += decode_char(char)
+  end
+  my_string
+end
```

Comment:
- [ ] When I run `ruby morse.rb` in the terminal, It gets an error. It says "undefined local variable or method". I think you can change `my_string` to `new_str` to fix the issue. :+1: